src/lib/http.service.ts

```diff
@@ -10,14 +11,19 @@ export class NGXLoggerHttpService {
 
   }
 
-  logOnServer(url: string, log: NGXLogInterface, customHeaders: HttpHeaders): Observable<any> {
+  logOnServer(url: string, log: NGXLogInterface, options: object): Observable<any> {
+    return this.http.post(url, log, options || {});
+  }
+
+  setHeaders(customHeaders?: HttpHeaders): HttpHeaders {
```

Comment:
`setHeaders` is confusing, since we're not really setting any headers here, we're extending our custom headers with the `Content-Type` header. I also don't think this should be in the `NGXLoggerHttpService`. Let's put this inline when we create the options object.